carbon-design-system/carbon-addons-iot-react
# fix(Translations & Regression): changes for translation an regression…

Related:  https://github.ibm.com/wiotp/Maximo-Asset-Monitor/issues/2521

**Summary**
Changes for translation bugs in monitor. Also one regression fix where callback was removed from bad merge in `CardEditForm.jsx`. Was purposely added [here](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3454/files) and accidentally removed [here](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3450/files)
- fix i18n strings in `DataSeriesFormContent.jsx`

**Change List (commits, features, bugs, etc)**

- add callback `onEditDataItem` back that was accidentally removed in a bad merge for `CardEditForm.jsx`. 
- add missing `translateWithId` callback to `ThresholdsFormItem.jsx`
- fix strings in `DataSeriesFormItemModal.jsx`

**Acceptance Test (how to verify the PR)**

- Go to [this](https://deploy-preview-3466--carbon-addons-iot-react.netlify.app/?path=/story/2-watson-iot-experimental-%E2%98%A2%EF%B8%8F-dashboardeditor--with-initial-value) story.
- Click on "Default rendered card" and then go to side panel and choose 'v2 pressure' from the data item drop down
- click edit on v2 pressure that opens modal. 
- see translated string. 

**Regression Test (how to make sure this PR doesn't break old functionality)**

- all the dashboard editor feature should work with out any issue?



**Things to look for during review**
